ouroboros-consensus/src/Ouroboros/Consensus/MiniProtocol/ChainSync/Server.hs

```diff
@@ -89,61 +92,85 @@ chainSyncBlocksServer tracer chainDB flr =
 --
 -- All the hard work is done by the 'Follower's provided by the 'ChainDB'.
 --
-chainSyncServerForFollower ::
-     forall m blk b.
-     ( IOLike m
-     , HasHeader (Header blk)
-     )
-  => Tracer m (TraceChainSyncServerEvent blk)
-  -> ChainDB m blk
-  -> Follower  m blk (WithPoint blk b)
-  -> ChainSyncServer b (Point blk) (Tip blk) m ()
-chainSyncServerForFollower tracer chainDB flr =
+chainSyncServerForReader
```

Comment:
Also, we recently renamed `Reader` to `Follower`. It looks like maybe your rebase lost that in this diff.